### DISCUSSION TITLE
Search in prior knowledge does not retrieve the right record

### DISCUSSION THREAD

**Discussion by StrootIris** [asreview/asreview#1876] [Q&A]
For our project, we have identified 10 previous records that are relevant and these we want to use as prior knowledge for training the model. 

When trying to label the relevant prior knowledge in search mode, I cannot retrieve any of the records I have in mind. I have used various combinations of keywords, title, authors and DOI, but nothing seems to work to get to the records. Instead, records that are irrelevant or show some match the search criteria are retrieved, but not the one I have in mind. I have checked if the records are included in the dataset, and that is the case so they should be available in the dataset. The entire dataset is 23k records and some show may show similarities with eachother, but the search terms I have used are as specific as possible to retrieve the record I have in mind. 

Can you help me with this issue, so that I can find the 10 records we want to use as prior knowledge?

**Comment by Rensvandeschoot** [comment]
ASReview uses fuzzy matching (details can be found [here](https://asreview.readthedocs.io/en/latest/generated/asreview.search.fuzzy_find.html#asreview.search.fuzzy_find)). Because of this, the search might not always return exact matches, which could be why the correct records are not appearing.

To work around this, you could directly label the records you want to use as prior knowledge within the dataset itself. If you’re working with an Excel file, add a column with one of the [recognizable column names](https://asreview.readthedocs.io/en/latest/data_format.html#tabular-file-format), such as "included" or "label_included." For each record add a `1` for relevant prior knowledge, a `0` for irrelevant prior knowledge, and leave blank if you want to screen the record. ASReview will automatically recognize this column and use the labels you provided as prior knowledge when starting the review.

**Comment by StrootIris** [comment]
Thank you for the fast reply. With the method you explained, I could work around the issue and was able to add the relevant prior knowledge.
